var/spack/repos/builtin/packages/py-fenics-basix/package.py

```diff
@@ -35,6 +35,7 @@ class PyFenicsBasix(PythonPackage):
     depends_on("py-numpy@1.21:", type=("build", "run"))
```

Comment:
setuptools is no longer required

Reply:
CMake 3.19+ required now

Reply:
@ma595 setuptools isn't required for `0@.8:`, but it is required for earlier versions.